applications/repertoire/templates/job-schema-update.yaml

```diff
@@ -0,0 +1,98 @@
+{{- $root := . -}}
+{{- if $root.Values.managedTapSchema }}
+{{- range $app, $config := .Values.config.tapServers }}
+{{- if $config.enabled }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "repertoire-{{ $app }}-schema-update"
+  annotations:
+    helm.sh/hook: "pre-install,pre-upgrade"
+    # helm.sh/hook-delete-policy: "hook-succeeded"
```

Comment:
Need to change to hook-succeeded before we merge in, I've left it to before-hook-creation for testing purposes and being able to see the logs.